dir-overlay/README.md

```diff
@@ -0,0 +1 @@
+See specifics for this Update Module at [Mender Hub](https://hub.mender.io/t/directory-overlay/)
```

Comment:
Should we have an AUTHOR note? I feel like contributors would like to be mentioned.

Reply:
Do not have a strong opinion, they are "mentioned" in the `git log` and if they have created a post on Mender Hub that will also contain information who authored it.

But based on your feedback I have created another "template" for the [README](https://github.com/mirzak/mender-update-modules/tree/readme-template-suggestion/dir-overlay) which contains a bit more information but also a "Maintainer" section. This does contain some duplication between Mender Hub and README but you can based on the README determinate if this module is for you, before you click on the Mender Hub link.

Reply:
Where is this template?

Reply:
> But based on your feedback I have created another "template" for the README 

"README" is a link, but here it is full

https://github.com/mirzak/mender-update-modules/tree/readme-template-suggestion/dir-overlay

Reply:
I know. But that commit should be part of the PR, right? That is what is confusing me

Reply:
No, I did not want to overwrite the "current template". Just a base for discussion.

Reply:
Once we agree on template I will of course update this PR

Reply:
Got it. The template looks good to me.

Reply:
Overall this is great stuff!
On the template, should we include some "raw" notes on how to use the module?

I understand this can be less maintainable due to the duplication in Mender Hub, but I was just thinking if I clone this repo and I know in general how update modules work it would be nice to just have an example CLI string to install the Update Module it on the device and generate an Artifact. The rest should be quite similar across Update Modules.

What do you think?

Reply:
I think it makes sense. I have updated the README with some "raw" notes on usage but with a link to Mender Hub for more details.